### PR TITLE
Gh517

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

Note: mathtools has this change in for some time.

### CODE REVIEW

File: base/doc/ltnews36.tex [hunk 1]
```diff
@@ -271,6 +271,41 @@ \subsection{Improve \pkg{l3docstrip} integration into \pkg{docstrip}}
 \section{Bug fixes}
 
 
+
+\subsection{Prevent \TeX{} from loosing a \cs{smash}}
+
+When \TeX{} is typesetting a fraction, it will rebox the material in
+either numerator or the denumerator depending on which is wider. If
+the repackaged part consists of a single box, that box gets new
+dimensions and if it was build using a \cs{smash} that effect vanishes
```
Comment (davidcarlisle) [inline base/doc/ltnews36.tex:280]: built

File: base/ltmath.dtx [hunk 2]
```diff
@@ -370,9 +370,39 @@
...
-  \setbox\z@\hbox{$\m@th#1{#2}$}\finsm@sh}
+  \setbox\z@\hbox{$\m@th#1{#2}$}%
+%    \end{macrocode}
+%    The empty brace groups in front of the smashed box (which is
+%    placed by \cs{finsm@sh}) ensures that a \cs{smash} in math is not
+%    just producing a single box  with its dimensions altered, but a
+%    box plus this second ord atom. The reason is that \TeX{} sometimes reboxes a
+%    box if its the only thing in a place like the denominator of a
+%    fraction. This would then undo the smashing and the addiitonal
+%    kern prevents that. Two ord atoms in a row do not alter the
```
Comment (davidcarlisle) [inline base/ltmath.dtx:389]: kern or atom?

**Review by davidcarlisle — APPROVED** [review]
a couple of comments on the doc, but otherwise looks good

File: base/ltmath.dtx (hunk 2 above)
Comment (Skillmon) [reply base/ltmath.dtx:389]: Also, wrongly placed i in "addiitonal" the line above.

Comment (FrankMittelbach) [reply base/ltmath.dtx:389]: yep, atom ... was initially a kern then changed

**Pull request merged**